On a fresh clone and checkout running the provided mvn command:

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.514 s
[INFO] Finished at: 2022-05-06T15:33:23-07:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.3.2:javadoc (default-cli) on project hauth: An error has occurred in Javadoc report generation:
[ERROR] Exit code: 1 - /home/avuong/ucla-workspace/test/src/main/java/edu/ucla/library/iiif/auth/handlers/AccessCookieHandler.java:19: error: cannot find symbol
[ERROR] import edu.ucla.library.iiif.auth.MessageCodes;
[ERROR]                                  ^
[ERROR]   symbol:   class MessageCodes
[ERROR]   location: package edu.ucla.library.iiif.auth
[ERROR] /home/avuong/ucla-workspace/test/src/main/java/edu/ucla/library/iiif/auth/handlers/AccessModeHandler.java:9: error: cannot find symbol
[ERROR] import edu.ucla.library.iiif.auth.MessageCodes;
[ERROR]                                  ^
[ERROR]   symbol:   class MessageCodes
[ERROR]   location: package edu.ucla.library.iiif.auth
[ERROR] /home/avuong/ucla-workspace/test/src/main/java/edu/ucla/library/iiif/auth/handlers/AbstractAccessTokenHandler.java:13: error: cannot find symbol
[ERROR] import edu.ucla.library.iiif.auth.MessageCodes;

--- a/src/main/java/edu/ucla/library/iiif/auth/services/DatabaseService.java
+++ b/src/main/java/edu/ucla/library/iiif/auth/services/DatabaseService.java
@@ -72,7 +72,7 @@ public interface DatabaseService {
     /**
      * Sets the given items.
      *
-     * @param aItems An array of objects that can be deserialized to a list of {@link DatabaseServiceImpl.Item}
+     * @param aItems An array of objects that conforms to the request body schema of the "postItems" OpenAPI operation
      * @return A Future that resolves once the items have been set
      */
     Future<Void> setItems(JsonArray aItems);

